bin/ares-install.js

```diff
@@ -148,10 +148,13 @@ function install() {
 function list(){
     installLib.list(options, function(err, pkgs) {
         let strPkgs = "";
-        if (pkgs instanceof Array) pkgs.forEach(function (pkg) {
-            if (!argv.type || (argv.type && (argv.type === pkg.type))) {
-                strPkgs += pkg.id + '\n';
+        if (Array.isArray(pkgs)) pkgs.forEach(function (pkg) {
```

Comment:
Please separate if and foreach

if (Array.isArray(pkgs)) {
pkgs.forEach(function (pkg) {

Reply:
Done.